hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/handle/SystemFileUpdateFacilityTest.java

```diff
@@ -179,9 +201,136 @@ void exchangeRateManagerUpdatedOnFileUpdate() {
         files.put(fileID, File.newBuilder().contents(FILE_BYTES).build());
 
         // when
-        subject.handleTxBody(state, txBody.build());
+        subject.handleTxBody(state, txBody.build(), new SingleTransactionRecordBuilderImpl(CONSENSUS_NOW));
 
         // then
         verify(exchangeRateManager, times(1)).update(FileUtilities.getFileContent(state, fileID));
     }
+
+    @Test
+    void handleThrottleFileTxBodyWithEmptyListOfGroups() {
```

Comment:
I'd say this functionality should be tested in `ThrottleManagerTest`, because `SystemFileUpdateFacility` does not do much more than rethrowing the `Exception`.